### PR TITLE
added input select to media_player

### PR DESCRIPTION
**Pull request by TheRealLink**

### CODE REVIEW

File: src/cards/ha-media_player-card.html [hunk 1]
```diff
@@ -138,6 +145,28 @@
             class='self-center'
           ></paper-icon-button>
         </div>
+        <div
```
Comment (balloob) [inline src/cards/ha-media_player-card.html]: This should not be added to the normal card, the more info card only.

File: src/more-infos/more-info-media_player.html [hunk 2]
```diff
@@ -64,6 +64,16 @@
           on-change='volumeSliderChanged' class='flex'>
         </paper-slider>
       </div>
+      <div class='controls layout horizontal justified' hidden$='[[!supportsSelectSource]]'>
+          <paper-icon-button icon="mdi:login-variant"></paper-icon-button>
```
Comment (balloob) [inline src/more-infos/more-info-media_player.html]: If you just want to render an icon, please use `iron-icon` instead.

File: src/more-infos/more-info-media_player.html [hunk 3]
```diff
@@ -64,6 +64,16 @@
...
+          <paper-dropdown-menu style="margin-left: 10px;" no-label-float selected-item-label='{{selectedSource}}' on-tap='stopPropagation'>
```
Comment (balloob) [inline src/more-infos/more-info-media_player.html]: Why did you add the `on-tap` event listener ? I would expect this to be handled by the `paper-dropdown` ?

Comment (balloob) [inline src/more-infos/more-info-media_player.html]: Please no inline CSS.

File: src/more-infos/more-info-media_player.html [hunk 4]
```diff
@@ -64,6 +64,16 @@
...
+            <paper-menu class="dropdown-content" selected="[[computeSelectedSource(stateObj)]]">
```
Comment (balloob) [inline src/more-infos/more-info-media_player.html]: This class does not seem to be used.

File: src/more-infos/more-info-media_player.html (hunk 3 above)
Comment (balloob) [inline src/more-infos/more-info-media_player.html]: Why not use `label="source"` attribute to render what the dropdown is about?

Comment (TheRealLink) [inline src/more-infos/more-info-media_player.html]: i used the input_select example as a template since im kinda new to this framework

File: src/more-infos/more-info-media_player.js [hunk 5]
```diff
@@ -186,4 +231,8 @@ export default new Polymer({
     serviceData.entity_id = this.stateObj.entityId;
     serviceActions.callService('media_player', service, serviceData);
   },
+
+  stopPropagation(ev) {
```
Comment (balloob) [inline src/more-infos/more-info-media_player.js]: Doesn't seem to be used

File: src/more-infos/more-info-media_player.html [hunk 6]
```diff
@@ -64,6 +72,16 @@
           on-change='volumeSliderChanged' class='flex'>
         </paper-slider>
       </div>
+      <div class='controls layout horizontal justified' hidden$='[[!supportsSelectSource]]'>
+          <iron-icon class="source-input" icon="mdi:login-variant"></iron-icon>
+          <paper-dropdown-menu class="source-input" no-label-float label='[[source]]'>
```
Comment (balloob) [inline src/more-infos/more-info-media_player.html]: We shouldn't use the label to show the current selected option. Instead it should say 'Source'.

Comment (balloob) [inline src/more-infos/more-info-media_player.html]: I actually like the label-float, it shows what the value that is selected is about. So it would show

source:
HDMI 1

**Comment by TheRealLink** [comment]
added floating label, removed unused code

**Comment by balloob** [comment]
The only style guide we have is the official material design guide lines: https://www.google.com/design/spec/material-design/introduction.html but it's all high level.
